pkg/lib/authn/authenticator/password/generator.go

```diff
@@ -1,5 +1,84 @@
 package password
 
+import (
+	"crypto/rand"
+	"encoding/binary"
+	"math"
+	"strings"
+)
+
+// Character list for each category.
+const (
+	CharListLowercase    = "abcdefghijklmnopqrstuvwxyz"
+	CharListUppercase    = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	CharListAlphabet     = CharListLowercase + CharListUppercase
+	CharListDigit        = "0123456789"
+	CharListAlphanumeric = CharListAlphabet + CharListDigit
+	// TODO: Define real list of special characters
+	CharListSymbol = "!@#$%^&*()-_=+,.?/:;{}[]`~"
```

Comment:
Found that we actually don't have a finite list of symbols. A character is considered symbol as long as it is not alphabet / digit, same on portal and server

https://github.com/authgear/authgear-server/blob/2b8ec9a5f009f6cd984e38926a551a48b0528346/pkg/lib/authn/authenticator/password/checker.go#L31-L34

Reply:
https://developer.apple.com/documentation/security/password_autofill/customizing_password_autofill_rules

Use the `special` class from here, but exclude the space.

Reply:
Updated in [228084f](https://github.com/authgear/authgear-server/pull/4535/commits/228084ffe1b50c9269a8ff1f459ee4379da341d7)